Earthfile

```diff
@@ -783,21 +785,25 @@ trivy-scan:
     SAVE ARTIFACT /build/results.json results.json AS LOCAL build/${ISO_NAME}-trivy.json
 
 grype:
-    FROM anchore/grype
+    ARG GRYPE_VERSION
```

Comment:
Loool we were not using the pinned version?

Reply:
vivere pericolosamente :D